components/ee/agent-smith/cmd/config-schema.go

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2021 Gitpod GmbH. All rights reserved.
-// Licensed under the Gitpod Enterprise Source Code License,
-// See License.enterprise.txt in the project root folder.
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
```

Comment:
Comment (non-blocking): In the root, there's only "License.AGPL.txt".

Reply:
Turns out this has always been wrong. Fixed now.